dateparser/parser.py

```diff
@@ -8,7 +8,7 @@
 from datetime import timedelta
 
 from dateparser.utils import set_correct_day_from_settings, \
-    get_last_day_of_month
+    get_last_day_of_month, get_previous_leap_year, get_next_leap_year
```

Comment:
Why don't write one method by line?, this way if we need add an import, just modify a line and avoid possible erros

Reply:
I don't see the point...

If you check the PEP8 guidelines, this is the normal way of using imports:
![imagen](https://user-images.githubusercontent.com/22377678/86565440-c93c4980-bfa2-11ea-9432-73c7f4a99c4f.png)
ref: https://www.python.org/dev/peps/pep-0008/#imports

Of course, I can use tuples to do multi-line imports, but that's normally used only when there are long names to be imported or heavy nested structures. 

In this case, I think that what I did is totally correct.

Reply:
@eamanu (comment above)